src/components/App.js

```diff
@@ -1,17 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-
-/* eslint-disable  no-unused-vars */
 import Calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <div id="calc" className="flex flex-col align-center">
-      <Display />
-      <ButtonPanel />
-    </div>
-  );
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { total: null, next: null, operation: null };
+    this.handleClickAPP = this.handleClickAPP.bind(this);
+  }
+
+  handleClickAPP(buttonName) {
```

Comment:
- [ ] [OPTIONAL] If you rewrite this function with the arrow function syntax, you won't need to bind it inside the constructor anymore. [Check this article about it.](https://medium.com/@joespinelli_6190/using-arrow-functions-to-avoid-binding-this-in-react-5d7402eec64)

Reply:
Agree, much better with funcitonal componets, but was a requirement to use a classe component for this APP component.